packages/ezjs_fetch/ezjs_fetch.0.3/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for Fetch"
+description: "Bindings for Fetch"
+maintainer: ["OCamlPro <contact@ocamlpro.com>"]
+authors: ["OCamlPro <contact@ocamlpro.com>"]
+license: "LGPL-2.1"
```

Comment:
It should be either:

```suggestion
license: "LGPL-2.1-only"
```

or

```suggestion
license: "LGPL-2.1-or-later"
```

I think it should be the latter because
> If the Library specifies a version number of this License which applies to it and "any later version", you have the option of following the terms and conditions either of that version or of any later version published by the Free Software Foundation.